app/src/main/java/org/systers/mentorship/viewmodels/RequestsViewModel.kt

```diff
@@ -39,7 +39,7 @@ class RequestsViewModel : ViewModel() {
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribeWith(object : DisposableObserver<List<Relationship>>() {
                     override fun onNext(relationsList: List<Relationship>) {
-                        allRequestsList = relationsList
+                        allRequestsList = relationsList.sortedWith(compareByDescending {it.sentOn})
```

Comment:
@anushaprem29 this should be the only change showing up here right?
Can you fix git history of this PR, to only show this change?